sites/platform/static/files/fetch/examples/relationships/opensearch

```diff
@@ -1,18 +1,22 @@
 {
     "username": null,
-    "scheme": "http",
-    "service": "opensearch12",
     "fragment": null,
     "ip": "169.254.99.100",
-    "hostname": "2e36wpnescmc5ffcddczsnhnai.opensearch12.service._.eu-3.platformsh.site",
-    "port": 9200,
     "cluster": "rjify4yjcwxaa-master-7rqtwti",
     "host": "opensearch.internal",
-    "rel": "opensearch",
     "path": null,
     "query": [],
     "password": null,
+    "port": 9200,
+    "host_mapped": false,
+    "service": "opensearch12",
+    "hostname": "2e36wpnescmc5ffcddczsnhnai.opensearch12.service._.eu-3.platformsh.site",
```

Comment:
please do not expose potential IPs ;) 

for such config, please use: 
- IP: 123.456.78.90
-  "hostname": "azertyuiopqsdfghjklm.opensearch12.service._.eu-1.platformsh.site",